tests/query/processors/test_mapping_promoter.py

```diff
@@ -5,29 +5,33 @@
 from snuba.datasets.schemas.tables import TableSource
 from snuba.query.expressions import Column, FunctionCall, Literal
 from snuba.query.logical import Query as LogicalQuery
-from snuba.query.processors.mapping_promoter import (
-    MappingColumnPromoter,
-    PromotedColumnsSpec,
-)
+from snuba.query.processors.mapping_promoter import MappingColumnPromoter
 from snuba.request.request_settings import HTTPRequestSettings
 
+columns = ColumnSet(
+    [
+        ("promtoed", Nullable(String())),
```

Comment:
I just noticed this is misspelled throughout this file:

```suggestion
        ("promoted", Nullable(String())),
```